modules/ROOT/pages/dynamic-expression-field.adoc

```diff
@@ -128,11 +128,10 @@ After you post the message, return to your flow in Design Center, open the log,
 
 === What to do next
 
-Try adding a Set Variable card after the HTTP Listener card, creating a variable that stores the value of `payload.firstName`, and then changing the message in the Logger to use the variable instead of the payload element. In the Logger card, the variable will appear in the list on the right. You can delete the payload element from the message and then drag in the variable.
+Try adding a Set Variable card after the HTTP Listener card, creating a variable that stores the value of `payload.firstName`, and then changing the message in the Logger to use the variable instead of the payload item. In the Logger card, the variable will appear in the list on the right. You can delete the payload item from the message and then drag in the variable.
```

Comment:
I think it would be valuable to say what the "Set Variable" card does.  Something like: "When data is returned from a connector, it is best practice to add 'set variable' after that connector in order to save that data for user later in the flow."

Reply:
Done.